mkt/developers/forms.py

```diff
@@ -253,12 +257,21 @@ def save(self, addon, commit=True):
                                    .replace('-', '/'))
                 if filetype in mkt.VIDEO_TYPES:
                     self.instance.update(filetype=filetype)
-                    vtasks.resize_video.delay(upload_path, self.instance.pk,
-                                              user_pk=mkt.get_user().pk)
+                    chain(
```

Comment:
I wonder if we should have wrappers for these `chain` calls in tasks. Maybe even rename the task to `_resize_video` and move the chaining to a wrapper that would be called `resize_video` ?